lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -261,10 +261,11 @@ defmodule ExUnit.DocTest do
           reraise e, stack
```

Comment:
Ah, ok. We convert compile errors into AssertionError that we raise internally. We should not remove it indeed!